src/selection/selection.js

```diff
@@ -472,18 +494,23 @@ class Selection {
 
   /**
    * Select all cells.
+   *
+   * @param {boolean} includeCorner `true` If the selection should include the corner header, `false` otherwise.
    */
-  selectAll() {
+  selectAll(includeCorner = false) {
     const nrOfRows = this.tableProps.countRows();
     const nrOfColumns = this.tableProps.countCols();
+    const startCoords = includeCorner ?
```

Comment:
Perf tip:
`startCoords` declaration can be moved after this: 
```js
if (nrOfRows === 0 || nrOfColumns === 0) {
  return;
}
```
It is not necessary to create `CellCorrds` instances when dataset is empty.